src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalAnalysisResult.js

```diff
@@ -211,11 +225,30 @@ const ClinicalAnalysisResult = ({
   overallSurvivalData,
   populateSurvivalData,
   survivalPlotLoading,
+  parsedFacets,
+  displayVariables,
+  clinicalAnalysisFields,
+  hits,
   ...props
 }: IAnalysisResultProps) => {
   const setName = Object.values(sets.case)[0];
   const setId = Object.keys(currentAnalysis.sets.case)[0];
 
+  if (hits.total === 0) {
+    return (
+      <Column style={{ margin: '2rem' }}>
+        <Row spacing={'10px'} style={{ alignItems: 'center', width: '80%' }}>
+          <Icon style={{ height: 50, width: 50 }} />
+          <h1 style={{ fontSize: '2.5rem', margin: 5 }}>Clinical Analysis</h1>
+        </Row>
+        <Row style={{ marginTop: '1rem', marginLeft: '1rem' }}>
+          Analysis is deprecated because {currentAnalysis.name} is a deprecated
```

Comment:
How about `{currentAnalysis.name} is a deprecated set.` or `There is no value in {currentAnalysis.name} analysis.`?

Reply:
We can ask Rosi and Brandon about precise language here. I was borrowing the basic idea from the other analysis types. But I think I should change it to `because {setName} is a deprecated set`